### ISSUE TITLE
Error message is shown when location (lat/long) are not defined

### ISSUE THREAD

**Issue by stephanj** [devoxx/MyDevoxxGluon#310]
Would be nice if the event is still shown even if the location lat/long are not defined.

This would mean that only the location map is not available and should show a message saying "location details (lat/long) are not provided".

![Screenshot_20220520-102454](https://user-images.githubusercontent.com/179457/170939813-ca2bd32d-2e8d-4a16-b783-b0bbf625400d.jpg)

**Comment by jperedadnr** [comment]
When lat/lon or location Address are null, 
```
May 30, 2022 1:22:15 PM com.devoxx.service.DevoxxService lambda$retrieveConference$18
WARNING: Remote function 'conference' failed.
java.lang.Exception: java.io.IOException: RemoteFunction failed with status code: 400.
        at java.base/java.lang.Thread.getStackTrace(Thread.java:1610)
        at com.gluonhq.connect.provider.DataProvider.retrieveObject(DataProvider.java:122)
        at com.gluonhq.cloudlink.client.data.RemoteFunctionObject.call(RemoteFunctionObject.java:50)
        at com.devoxx.service.DevoxxService.retrieveConference(DevoxxService.java:414)
        at com.devoxx.views.cell.ConferenceCell.lambda$updateItem$6(ConferenceCell.java:184)
...
Caused by: java.io.IOException: RemoteFunction failed with status code: 400.
        at com.gluonhq.impl.cloudlink.client.data.function.GluonCloudRemoteFunctionObjectDataReaderImpl.readObject(GluonCloudRemoteFunctionObjectDataReaderImpl.java:98)
        at com.gluonhq.connect.provider.DataProvider.lambda$retrieveObject$11(DataProvider.java:125)
 
 ```
The RemoteFunction `conference` fails:

```
Caused by: java.lang.ClassCastException: class javax.json.JsonValueImpl cannot be cast to class javax.json.JsonNumber (javax.json.JsonValueImpl and javax.json.JsonNumber are in unnamed module of loader 'app')
        at org.glassfish.json.JsonObjectBuilderImpl$JsonObjectImpl.getJsonNumber(JsonObjectBuilderImpl.java:247)
        at com.gluonhq.devoxx.serverless.util.ConferenceUtil.createCleanResponseForClientFromNewEndpoint(ConferenceUtil.java:67)
```
and points to [line 67](https://github.com/devoxx/MyDevoxxGluon/blob/gluonfx/DevoxxRemoteFunctions/src/main/java/com/gluonhq/devoxx/serverless/util/ConferenceUtil.java#L67):
```
        builder.add("venueLongitude",   conf.getJsonNumber("venueLongitude"));
        builder.add("venueLatitude",    conf.getJsonNumber("venueLatitude"));

```

**Comment by jperedadnr** [comment]
Fixed with #314
